tool/tsh/common/app.go

```diff
@@ -470,6 +471,25 @@ func formatAppConfig(tc *client.TeleportClient, profile *client.ProfileStatus, r
 	}
 }
 
+// initClusterAndProfile initializes a cluster client and also returns the
```

Comment:
`initClusterAndProfile` sounds like a factory method that creates a cluster. maybe we can improve the name a little